CMakeLists.txt

```diff
@@ -85,6 +87,8 @@ link_libraries (
     ${PYTHON_LIBRARIES}
     ${Boost_LIBRARIES}
     ${Qt5Widgets_LIBRARIES}
+    ${FFMPEG_LIBRARIES}
+    swresample
```

Comment:
I remember that I had problem with linking on Ubuntu without these lines but I'm not sure for now, I'll check it.